backend/pyproject.toml

```diff
@@ -6,6 +6,8 @@ readme = "README.md"
 requires-python = ">=3.13"
 dependencies = [
     "fastapi-csrf-protect>=1.0.3",
+    "asyncio>=3.4.3",
```

Comment:
```suggestion
```
Last nitpick, this is built-in module. `uv.lock` file needs to be committed again, after `uv sync` is run.

Reply:
Actually locking and syncing might be obsolete, when removal is done with `uv`. I had in mind, that it would need to be synced, because `pyproject.toml` would be changed manually.